docs/widgetbook-cloud/reviews.mdx

```diff
@@ -1,124 +1,58 @@
-
-Once a review's changes are approved, the developer closes the Pull-Request, updating the review status to `merged`.
-
-## How to set up addons and knobs
-
-In order for the addons and knobs in your Widgetbook builds to work with our review feature, make sure you have the `WidgetbookCloudIntegration` added to the `integrations` array in your `Widgetbook` app:
-
-```dart
-@widgetbook.App()
-class WidgetbookApp extends StatelessWidget {
-  const WidgetbookApp({super.key});
-
-  @override
-  Widget build(BuildContext context) {
-    return Widgetbook.material(
-      // Use the generated directories variable
-      directories: directories,
-      addons: [],
-      integrations: [
-        // Important to send addons & knobs to Widgetbook Cloud
-        WidgetbookCloudIntegration(),
-      ],
-    );
-  }
-}
-```
-
-## How to add a Figma design link
-
-To add a Figma design link, you need to provide the `designLink` property to a use case:
-1. Got to Figma
-1. Go to the component that resembles your Widget
-1. Right-mouse click on the component
-1. Select `Copy/Paste as` > `Copy Link`
-1. Set the `designLink` property of the `@UseCase` annotation by pasting the copied link
-1. Let the re-run `build_runner` 
-1. Commit the changes to Git
-
-```dart
-// Example from https://github.com/widgetbook/groceries-demo/blob/chore/design-link-for-appbar/lib/core/app_bar.dart
-@UseCase(
-  designLink: 'https://www.figma.com/file/EXuEpwiyksLAejYX1qr1v4/Fluttercon-Berlin-2023-Demo?type=design&node-id=277-3056&mode=design&t=nVL8hLmc1jlcilZL-4',
-  name: 'Default',
-  type: AppBar
-)
-```
+| Status   | Description                                             |
+| -------- | ------------------------------------------------------- |
+| Open     | Awaiting assessment.                                    |
+| Merged   | Linked with merged PRs, thus considered merged.         |
+| Closed   | Associated with closed PRs, hence viewed as closed.     |
+| Updating | Missing crucial information for an open classification. |
+
+## Widgetbook x Figma
+
+To have "View in Figma" button in your reviews, you need to add the Figma URL to your use-cases:
+
+1. In you Figma design file, navigate to the component that resembles your Widget.
+1. Copy the link to the component by right-mouse clicking on the component and selecting `Copy/Paste as` > `Copy Link`.
+1. Set the `designLink` property of the `@UseCase` annotation by pasting the copied link.
+
+   ```dart
+   // Example from https://github.com/widgetbook/groceries-demo/blob/main/widgetbook/lib/core/app_bar.dart
+   @UseCase(
+     designLink: 'https://www.figma.com/file/EXuEpwiyksLAejYX1qr1v4/Fluttercon-Berlin-2023-Demo?type=design&node-id=277-3056&mode=design&t=nVL8hLmc1jlcilZL-4',
+     name: 'Default',
+     type: AppBar
+   )
+   ```
+
+1. Re-run `build_runner` to update the use-cases metadata.
+
+   ```bash
+   dart run build_runner build -d
+   ```
```

Comment:
I would information on how we suggest to review:
1. Check if your visual changes are intentional or not
2. If all changes are intentional, ask team mates for review
3. To provide feedback, copy the link to the Widgetbook Cloud Review Story and leave a comment in the GitHub PR. Note: If the reviewer doesn't have access to GitHub (e.g. a designer), we suggest using messaging tools, like Slack. Please reach out to us in case this doesn't suit you and you want to use a dedicated comment feature in Widgetbook Cloud.